frontend/src/components/5.Lobby/Lobby.js

```diff
@@ -106,8 +106,8 @@ const Lobby = () => {
           <div className='MakeTextCentre'>
             <h2> Share the group code:</h2>
             <h1>{socketContext.code}</h1>
-            <a href={'https://yumble.xyz/JoinGroup/'}>
-            https://yumble.xyz/JoinGroup
+            <a href={'https://yumble.xyz'}>
+            https://yumble.xyz
```

Comment:
I think this could be changed to a copy to clipboard button (I think it's used elsewhere) as currently the link doesn't do much just directs you to the homepage.

Reply:
yea but since the routing changed, https://yumble.xyz/JoinGroup does not do anything and will go back to startpage. So the user will have to press 'Join Group' button anyways.

Reply:
And if we do do that, Won will need o change his part which I'm not sure we have enough time for tmr.

Reply:
Okay fair - I'll unlink the share button ticket link so we can revisit it after the demo